Adding BigQuery + StackDriver Monitoring API Showcase sample.

dzlier-gcp opened the pull request:
After running some queries on the most often paired API's, I found that bigquery-json and monitoring are among the top paired in java projects. This version of the API showcase includes aspects of both the bigquery SimpleApp and the monitoring QuickStart, demonstrating how to incorporate both into a single project.

In this case, after making the call to BigQuery, the app adds metrics about how long it took the query to process, and the number of rows returned.

This tutorial will be linked to as the second part of the new [/java/docs](https://cloud.google.com/java/docs) page, after a similar link to the Spring Boot on AppEngine Standard tutorial.

kurtisvg commented on bigquery/cloud-client/src/test/java/com/example/bigquery_logging/SimpleAppIT.java:
```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2016 Google Inc.
```
2018

kurtisvg commented on bigquery/cloud-client/README.md:
```diff
@@ -46,3 +46,45 @@ documentation](https://cloud.google.com/bigquery/create-simple-app-api):
+	
+Provide the projectId as a system variable when running the sample.
+
+    mvn exec:java -Dexec.mainClass=com.example.bigquery_logging.SimpleApp -DprojectId=<your-project-id>
```
It looks like this is an appengine product - we generally suggest using maven to run/delpoy, ie `mvn appengine:run` and `mvn appengine:deploy`

dzlier-gcp commented:
Updated the location, README instructions, and changed it from a command line program to an app with a web index page and simple forms displaying run results/metrics.

dzlier-gcp commented:
And ran through the checkstyle warnings/errors.

kurtisvg commented:
@dzlier-gcp One of the tests is failing on a null pointer exception. 

Cloud you also add this to the README.md?
```markdown
<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=appengine-java8/bigquery/README.md">
<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
```
